sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -470,6 +473,16 @@ public static partial class MultipartResponse
         public static Azure.Response[] Parse(Azure.Response response, bool expectCrLf, System.Threading.CancellationToken cancellationToken) { throw null; }
         public static System.Threading.Tasks.Task<Azure.Response[]> ParseAsync(Azure.Response response, bool expectCrLf, System.Threading.CancellationToken cancellationToken) { throw null; }
     }
+    public partial class PipelineContext
+    {
+        internal PipelineContext() { }
+        public System.DateTimeOffset CreatedOn { get { throw null; } }
+        public int RetryAttempt { get { throw null; } set { } }
```

Comment:
RetryAttempt value is available downstream from RetryPolicy in the pipeline, for use in GeoRedunant policy implementations.

Reply:
Could we call it RetryCount? Also, I assume it starts with the value of zero, i.e. no retries attempted yet.

Reply:
In the approach I proposed, I had this called AttemptNumber and it starts as 1. I think this is a little bit more intuitive, but am open to other opinions.